package.json

```diff
@@ -18,18 +18,11 @@
     "postpublish": "git push origin && git push origin --tags",
     "build": "rimraf dist && tsc -p tsconfig.json && tsc -p tsconfig.esm.json",
     "pretest": "yarn build",
-    "test": "tape dist/cjs/test/{unit,compat}/*.js",
+    "test": "tape dist-test/cjs/test/{unit,compat}/*.js",
```

Comment:
I preferred my way which, while requiring an extra dependency, did not require building before testing. With setup like this while testing your pull request on your machine you may forget to `pnpm build` before `pnpm test`.